### PR TITLE
Feature: Use listr to display progress and errors for transformations

### PR DESCRIPTION
**Pull request by okonet**
**What kind of change does this PR introduce?**
feature

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
N/A

**Summary**

This PR introduces better progress and error messages for failing transforms.

2017-03-19 at 14 55

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by pksjce** [comment]
Hey @okonet - I tried out the `migrate` command on this branch. It showed " ✔ Migrating config from v1 to v2" but not the substeps, like your screenshot.
Also, conflicts 👀

**Comment by okonet** [comment]
@pksjce I'll take a look at conflicts. Re: not showing sub-steps: it's just too damn fast :) It will show those if it would fail though. I'll play around with it a bit more to make it even friendlier.

**Comment by pksjce** [comment]
Shall I merge this now? It looks like things can be built on top of this?

**Comment by okonet** [comment]
Please do a review and approve/reject it and I'll merge it.

File: lib/migrate.js [hunk 1]
```diff
@@ -1,36 +1,83 @@
...
+	const tasks = new Listr([
+		{
+			title: 'Reading webpack config',
+			task: (ctx) => new PLazy((resolve, reject) => {
+				fs.readFile(currentConfigPath, 'utf8', (err, content) => {
+					if (err) {
+						reject(err);
+					}
+					try {
+						console.time('Reading webpack config');
```
Comment (okonet) [inline lib/migrate.js]: Ouch that shouldn't have made through the review :( Will remove it.

**Pull request merged**